indico/modules/events/abstracts/templates/abstract/public.html

```diff
@@ -232,6 +232,46 @@
         </div>
 {% endmacro %}
 
+{% macro _render_judgment_action(abstract) %}
+    {% set action = abstract.state.name %}
+    {% if abstract.can_judge(session.user) %}
+        {% set judge_name = abstract.judge.name %}
+        {% if action == 'withdrawn' %}
+            {% trans name=judge_name -%}
+                <strong>{{ name }}</strong> withdrew this abstract
```

Comment:
Should we maybe log who withdrew the abstract? It should generally be the submitter or one of the authors and accountability would not harm here, especially after abstract merging.